data/json/uncraft/comestibles/fruit.json

```diff
@@ -35,18 +35,186 @@
     "result": "irradiated_watermelon",
     "type": "uncraft",
     "activity_level": "LIGHT_EXERCISE",
-    "copy-from": "watermelon"
+    "time": "1 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "watermelon_wedge", 16 ] ] ]
   },
   {
     "result": "irradiated_melon",
     "type": "uncraft",
     "activity_level": "LIGHT_EXERCISE",
-    "copy-from": "melon"
+    "time": "1 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "melon_chunks", 4 ] ] ]
   },
   {
     "result": "irradiated_pineapple",
     "type": "uncraft",
     "activity_level": "LIGHT_EXERCISE",
     "copy-from": "pineapple"
+  },
+  {
+    "result": "pumpkin",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "pumpkin_cut", 4 ] ], [ [ "seed_pumpkin", 6 ] ] ]
+  },
+  {
+    "result": "irradiated_pumpkin",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "pumpkin_cut", 4 ] ] ]
+  },
+  {
+    "result": "rose_hips",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "30 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "seed_rose", 1 ] ] ]
+  },
+  {
+    "result": "grapes",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "30 s",
+    "components": [ [ [ "seed_grapes", 1 ] ] ]
+  },
+  {
+    "result": "tomato",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "30 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "seed_tomato", 1 ] ], [ [ "tomato_cut", 1 ] ] ]
+  },
+  {
+    "result": "mayapple",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "30 s",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "seed_mayapple", 1 ] ] ]
+  },
+  {
+    "result": "mulberries",
+    "type": "uncraft",
+    "activity_level": "NO_EXERCISE",
+    "time": "30 s",
+    "components": [ [ [ "seed_mulberries", 1 ] ] ]
+  },
+  {
+    "result": "zucchini",
```

Comment:
Another one, just a nitpick here, but for future easy of navigation it'd be best to have fruit in the fruit disassembly file and make a separate one for vegetables. This is gonna get confusing quickly otherwise.

Reply:
I was going by the botanical definition, IE a fruit is the seed-bearing part of a plant which is produced from the flower. By that definition, all of these items are fruits. If we want to use the colloquial definition of sweet=fruit, I can move some items to a new file, but that's not always going to be an objective way to organize them. What do you think?

Reply:
> I was going by the botanical definition, IE a fruit is the seed-bearing part of a plant which is produced from the flower. By that definition, all of these items are fruits. If we want to use the colloquial definition of sweet=fruit, I can move some items to a new file, but that's not always going to be an objective way to organize them. What do you think?

Personally in my opinion we should simply be going by their organization in json files. Of course, a 'vegetable' is really a colloquial term here, but for the most part things in that file add up with what's commonly regarded as a vegetable (I think the only outlier there that is not considered a veggie by 100% of the people I met is the tomato). Having them split in two files in the item json and one in disassemble will get confusing.

Reply:
But what about cucumbers! 😅

I made very little effort to organize the item and recipe jsons either - they were already kind of messy and in lumping similar items together, I probably made it worse. Since you brought it up, I'll sort all those tonight, including the uncrafts, with fruits and veggies sorted according to the colloquial definitions.